README.md

```diff
@@ -14,7 +14,7 @@ Here is a screenshot of the Jupyter XBlock in action:
 
 ## Usage
 
-Install this xblock with [Tutor](https://docs.tutor.overhang.io/) (Palm release):
+Install this xblock with [Tutor](https://docs.tutor.overhang.io/) (Quince release):
```

Comment:
Can you please update the instructions to explain that this xblock is automatically installed by the tutor-jupyter plugin? We should refer users  to the plugin docs.